### ISSUE TITLE
False positive for too-many-function-args with collections.Counter

### ISSUE THREAD

**Issue by pylipp** [PyCQA/pylint#3500]
### Steps to reproduce
1. Install Python 3.8.2 on Linux.
2. `pip install pylint==2.3.1`
3. Copy content to `file.py`:
```
"""module docstring"""
import collections

def main():
    """Docstring"""
    counter = collections.Counter()
    return counter


if __name__ == "__main__":
    main()
```
4. Run `pylint file.py`.

### Current behavior
`file.py:6:14: E1121: Too many positional arguments for constructor call (too-many-function-args)`

### Expected behavior
No errors; 10/10.

### pylint --version output
```
pylint 2.3.1
astroid 2.3.3
Python 3.8.2 (v3.8.2:7b3ab59, Apr  3 2020, 11:44:23)
[GCC 5.4.0 20160609]
```

The issue does not occur with the preview release (pylint 2.4.4).

**Comment by PCManticore** [comment]
Hey @pylipp Thanks for the report. I can't reproduce this with 2.4.4 which is already released! This means it was probably already fixed in 2.4.X, so you should upgrade when you get a chance. The preview release is going to be 2.5.0 though.

**Comment by pylipp** [comment]
Thanks for the support! The issue does not occur with 2.5.0
